types/index.d.ts

```diff
@@ -37,15 +37,15 @@ interface ChartGuide {
         max?: number;
         label?: string | { text: string; padding: number; };
         tickPeriod?: string;
-        tickFormat?: string;
+        tickFormat?: (x: any) => string | string;
```

Comment:
`(x: any) => string | string` is `(x: any) => string`.

Use `((x: any) => string) | string`